EnglishFilter/sections/antiadblock.txt

```diff
@@ -594,6 +594,8 @@ uaclips.com,kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! 
+codedosa.com#$#body * { visibility: visible!important; }
```

Comment:
What is this rule for?

Reply:
To prevent anti-adblock script.

Reply:
When AG is on:
![scre1](https://user-images.githubusercontent.com/37463155/128656291-1377cef6-db4e-4965-a860-be984905a7f9.png)
When AG is off:
![scre2](https://user-images.githubusercontent.com/37463155/128656294-d1f34b65-7e24-4c3f-9ec9-42b4b873d930.png)

Reply:
Not reproduced on my end. What's caused this? Please write steps to reproduce.

Reply:
Filter List:
AdGuard Base, AdGuard Mobile Ads, AdGuard URL Tracking, AdGuard Tracking Protection, Fanboy's Anti-Facebook List, Fanboy's Anti-thirdparty Fonts, AdGuard Social Media, AdGuard Annoyances, Online Malicious URL Blocklist, Spam404, NoCoin Filter List, AdGuard DNS, AdGuard Turkish, AdGuard French, AdGuard Russian, AdGuard Chinese, AdGuard German, AdGuard Japanese, AdGuard Dutch, AdGuard Spanish/Portuguese

How to reproduce:
1. Just visit https://codedosa.com/1602
2. Then, you can see the blank page.

Reply:
@zloyden

Reply:
Don't need to mention me, if you leave a comment. We see this in our email.
Btw, next time please provide a filter or rule that causes the problem.
Here it's caused by `Fanboy's Anti-thirdparty Fonts` -> `||googleapis.com/ajax/libs/webfont/$third-party`.
You don't have to fix it - I will report it to their author. Please remove the rule.

Reply:
Okay.
https://github.com/AdguardTeam/AdguardFilters/pull/90429/commits/24c672b38a3cd573d1373a54d1954d31ed219417

Reply:
Well done.